pillar/top.sls.in

```diff
@@ -26,6 +26,11 @@ metalk8s-{{ version }}:
     - match: compound
     - metalk8s.roles.minion
 
+  # Specific case for ca as we need signing_policy at the very beginning
```

Comment:
Not fond of this. Can't we inject the CA role in the pillar when needed?

Reply:
But how ? Because I tested in `_pillar/metalk8s.py` it's ok but we also need the version in `top.sls` so it's weird to set the version hardcoded in an ext_pillar.

Also should we really stick with `ca` role in the kubernetes node object as we only want one CA, so we should maybe only keep `metalk8s.ca.minion` no ?

Reply:
There's no `ca` *role* on the Node object, right? There's only `metalk8s.ca.minion` from static config.

Reply:
Ah, you meant, to match on that pillar value for the node name, instead of using the `role`. Sure, that may work. But then we'll still require a version-match somehow.

Reply:
For `bootstrap` roles we always include the pillar no matter the version so maybe it's ok to do the same for CA as it's only signing policies (or at least for the moment, because I don't know how we can have the version at this point)

Reply:
Following the discussion we had, lets keep it like this, I will just do some test with multiple pillarenv.